### PR TITLE
feat: Add Favorite Buttons to New Ideas

### LINKED ISSUE #13: Add Favorite Buttons
- A star button should be visible on all idea cards now that match the comp.
- When an idea is created, it should start not favorited.
- The star will toggle between orange and white to show the user whether that idea is favorited or not, so the CSS could be addressed in this ticket.
  - orange: favorited
  - white: not favorited

Note: The JS functionality for this should be addressed in https://github.com/ericahagle/ideabox/issues/14

Image

### PR DESCRIPTION
**Pull request by ericahagle**
# Description
- [x] A star button is visible on all idea cards now that match the comp.
- [x] When an idea is created, it defaults to not favorited.
- [x] Both white and orange assets are added to the repo, and referenced in the code.

# Contributors
@ericahagle 

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My PR has a clear description of all proposed changes.
- [x] My PR denotes any/all team members who contributed to it.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] feat: My code is clean and readable, so my teammates can easily review it.

# Notes
- Originally, I was going to have both the add of the buttons and the functionality in this PR, but I think it'll be better to just get the buttons in and open a new PR for the functionality.
- Both assets have been added and named, but there's no toggling between them yet. For now, the orange is just hidden.

### CODE REVIEW

**Comment by ericahagle** [comment]
I've got this done part of the way, but I'm still trying to get the toggling working and DOM updated.

**Review by butlertree — COMMENTED** [review]
I looks good.  We can add the functionality you mentioned in another ticket.

**Review by butlertree — APPROVED** [review]

File: styles.css [hunk 1]
```diff
@@ -160,16 +168,24 @@ textarea#title {
 
 .idea-btn-container {
   display: flex;
+  display: inline-grid;
```
Comment (ben-rosner-williamsburg) [inline styles.css:170]: @ericahagle Do we have two display properties here?

**Review by ben-rosner-williamsburg — APPROVED** [review]
Looks good to me now! Approved.

File: styles.css (hunk 1 above)
Comment (ericahagle) [reply styles.css:170]: 100% you were right. Fixed it!

**Pull request merged**